site/app/controllers/admin/SqlToolboxController.php

```diff
@@ -24,7 +24,19 @@ class SqlToolboxController extends AbstractController {
      * @Route("/courses/{_semester}/{_course}/sql_toolbox", methods={"GET"})
      */
     public function showToolbox(): WebResponse {
-        return new WebResponse(SqlToolboxView::class, 'showToolbox');
+        $this->core->getCourseDB()->query("SELECT * FROM information_schema.columns WHERE table_schema='public'");
+        $tables = $this->core->getCourseDB()->rows();
+        $organizedTables = [];
+        //Loop through and create a 2d array that holds all columns for each table name.
+        foreach ($tables as $table) {
+            if (!isset($organizedTables[$table['table_name']])) {
+                $organizedTables[$table['table_name']] = [];
+            }
+            array_push($organizedTables[$table['table_name']], $table['column_name']);
```

Comment:
Better to use:

```suggestion
            $organizedTables[$table['table_name']][] = $table['column_name'];
```

as this avoids a function call (`array_push`) that is unnecessary.